packages/gatsby-transform-url/src/index.ts

```diff
@@ -0,0 +1,148 @@
+import ImgixClient from 'imgix-core-js';
+import { parseHost, parsePath } from './common/uri';
+import {
+  IGatsbyImageFixedData,
+  IGatsbyImageFluidData,
+  IImgixParams,
+} from './types';
+
+const VERSION = '1.0.0-beta.2';
```

Comment:
Yeah, so `require`-ing from `package.json` doesn't "just work" in the browser. This requires either a Webpack or a Babel plugin to make this work. Since we don't use either of these, we can't be sure that this would work in the browser, so at this stage it has to stay like this.